lib/wab/impl/data.rb

```diff
@@ -312,8 +310,6 @@ def fix_value(value)
           }
         elsif Array == value_class
           value = value.map { |v| fix_value(v) }
-        elsif WAB::Utils.pre_24_fixnum?(value)
-          # valid value
```

Comment:
The check for pre 2.4 was after the Hash and Array for performance reasons. I guess at this point I'm caring less about that now that the embedded Ruby is so much faster.